newIDE/app/src/EventsSheet/EventsTree/Renderers/CommentEvent.js

```diff
@@ -71,6 +71,10 @@ export default class CommentEvent extends React.Component<
       }
     );
   };
+  
+  componentDidMount() {
+    this.edit();
```

Comment:
Are you sure about this? It sounds like you will start editing the field as soon as the component is rendered on screen, so if I scroll through events, they will start editing.

Reply:
I can confirm that it won't work.
Moreover, this has been fixed by #4846